kube/completer.go

```diff
@@ -13,45 +15,65 @@ import (
 )
 
 func NewCompleter() (*Completer, error) {
-	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
-	loader := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
-		loadingRules,
-		&clientcmd.ConfigOverrides{},
-	)
-
-	config, err := loader.ClientConfig()
-	if err != nil {
-		return nil, err
-	}
-
-	namespace, _, err := loader.Namespace()
-	if err != nil {
-		return nil, err
-	}
-
-	client, err := kubernetes.NewForConfig(config)
-	if err != nil {
-		return nil, err
-	}
-
-	namespaces, err := client.CoreV1().Namespaces().List(metav1.ListOptions{})
```

Comment:
It looks enough just polling this function call.

Reply:
Seems also possible, I tried to modify it again.